### PR TITLE
Make header sticky

### PR DESCRIPTION
**Pull request by henriettemoe**
- Making header sticky
- Separating header content and bottom content (bottom content now in SubHeader)
- Adding underline for the active language in the header

### CODE REVIEW

File: h5p-bildetema/src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -3,6 +3,9 @@
 .header {
   background-color: $burgundy;
   padding: $spacing--32 0;
+  position: sticky;
+  top: 0;
+  z-index: 2;
```
Comment (boyum) [inline h5p-bildetema/src/components/Header/Header.module.scss:8]: Perhaps we should move this to the list of z-indices in `common/abstracts/_variables.scss` to easier keep track of them.

Comment (henriettemoe) [reply h5p-bildetema/src/components/Header/Header.module.scss:8]: Yes!

File: h5p-bildetema/src/components/Header/Header.stories.tsx [hunk 2]
```diff
@@ -47,9 +37,9 @@ const Template = (isWordView: boolean): JSX.Element => {
...
 
 export const Default = (): JSX.Element => {
-  return Template(false);
+  return Template();
 };
 
 export const isWordView = (): JSX.Element => {
-  return Template(true);
+  return Template();
 };
```
Comment (boyum) [inline h5p-bildetema/src/components/Header/Header.stories.tsx:45]: Can we remove this, as it seems identical to the `Default`?

Comment (henriettemoe) [reply h5p-bildetema/src/components/Header/Header.stories.tsx:45]: Yes!

**Review by boyum — APPROVED** [review]

**Review by StianHaugland1 — APPROVED** [review]

**Pull request merged**